src/main/java/racing/generator/RandomNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package racing.generator;
+
+import java.util.Random;
+
+public class RandomNumberGenerator implements NumberGenerator{
+    private final int MAX_VALUE = 10;
+    private final Random random;
```

Comment:
```suggestion
    private static final Random RANDOM = new Random();
```

생성할 때마다 생성하는 것도 좋으나 이렇게 표현해도 좋습니다